### DISCUSSION TITLE
Change the size of the typo of avatar s name ?

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3048] [General Questions & Discussion] [answered]
Hello 

Where can i change the name of  typo size in a hub vloud custom ? When there are too many user closed   we can t see name of the avatar .
Thx

**Comment by robinkwilson** [comment, marked as answer]
Here's where the nametag's scale is defined: https://github.com/mozilla/hubs/blob/041dc57b33f07c3983e3a4897ff702263131a96e/src/hub.html#L120

Update this and you change the size! 

If you'd like to do this dynamically, you can select the element this way https://github.com/mozilla/hubs/blob/49ca6d091554989220a597780eb2ef9dfea87dc0/src/components/player-info.js#L128 and update the scale attribute.
